concent_api/core/transfer_operations.py

```diff
@@ -107,95 +107,92 @@ def store_pending_message(
 
 
 def create_file_transfer_token_for_concent(
-    report_computed_task: message.tasks.ReportComputedTask,
-    operation: FileTransferToken.Operation,
-    should_add_source: bool = False,
+    subtask_id: str = None,
```

Comment:
Please add `Optional`.

Reply:
Done just to these parameters which can be None.